bin/GameIndex.yaml

```diff
@@ -33363,15 +33363,15 @@ SLUS-20586:
   region: "NTSC-U"
   compat: 5
 SLUS-20587:
-  name: "Driver 3"
+  name: "Driv3r"
   region: "NTSC-U"
   compat: 5
 SLUS-20588:
   name: "Activision Anthology"
   region: "NTSC-U"
   compat: 5
 SLUS-20590:
-  name: "Spy Hunter 2"
+  name: "SpyHunter 2"
```

Comment:
It's two words https://lh3.googleusercontent.com/proxy/E1PTf4Tb3ZZ5QL73RbvgtsTco3k21aKpqgQLGqSlKLcl9TDAf1spNMzHC6ZixQsjQ8_uV-rAb50WvuZWK47LMCHbOkTujoh4yQxVGrmwGa1BcDrycwA5rI621ekVGTAkoA3DCVDxI5qpi8aaVJxENT5kEdu9cA